container/webapp.D

```diff
@@ -1,8 +1,22 @@
 FROM ghcr.io/autoreduction/base
 
+RUN python3 -m pip install --user --no-cache-dir gunicorn
+
+# Installs frontend from your local repository
+ADD . .
+RUN python3 -m pip install --user --no-cache-dir .
+
+# Uncomment below if you want to install from pypi
 # add --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple for packages in test.pypi.org
-# also when updating the version here make sure it matches the one in setup.py
-RUN python3 -m pip install --user --no-cache-dir gunicorn autoreduce-frontend==22.0.0.dev17
+# RUN python3 -m pip install --user --no-cache-dir autoreduce-frontend==22.0.0.dev21
+
+# Add fixtures to the project for dev/debugging
+ADD --chown=isisautoreduce:isisautoreduce autoreduce_frontend/autoreduce_webapp/fixtures/ fixtures/ 
+
+USER root
+RUN mkdir -p /var/www/api/
```

Comment:
What is this directory for?

Reply:
It's a shared folder between the frontend and nginx sidecar to store and serve static content